pkg/yamlmeta/node.go

```diff
@@ -57,8 +58,8 @@ func (m *Map) SetValue(val interface{}) error {
 }
 
 func (mi *MapItem) SetValue(val interface{}) error {
```

Comment:
looks like there is couple places where SetValue isnt checked for error return (in overlay pkg). probably worth throwing it in as well.

Reply:
Thanks for looking into that. I have a (apparently false) sense of security that we do a good job of error checking.

Reply:
it's typically good. i just like to double check.

Reply:
I just created a targeted inspect profile in my IDE to keep a pulse on this. There's one other place that's suspicious, will take a look after getting this PR landed.

https://github.com/vmware-tanzu/carvel-ytt/blob/01e2421ff04a2fa22c634b7cb3eedf04e054d7aa/pkg/template/core/go_value.go#L108